--- a/components/download/public/common/download_features.cc
+++ b/components/download/public/common/download_features.cc
@@ -9,13 +9,7 @@
 namespace download {
 namespace features {
 
-BASE_FEATURE(kParallelDownloading,
-#if BUILDFLAG(IS_ANDROID)
-             base::FEATURE_ENABLED_BY_DEFAULT
-#else
-             base::FEATURE_DISABLED_BY_DEFAULT
-#endif
-);
+BASE_FEATURE(kParallelDownloading, base::FEATURE_ENABLED_BY_DEFAULT);
 
 #if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC)
 BASE_FEATURE(kBackoffInDownloading, base::FEATURE_DISABLED_BY_DEFAULT);
