--- a/ui/native_theme/features/native_theme_features.cc
+++ b/ui/native_theme/features/native_theme_features.cc
@@ -37,7 +37,7 @@ BASE_FEATURE(kFluentScrollbar, base::FEA
 // TODO(crbug.com/40280779): Right now this feature flag will force Fluent
 // overlay scrollbars on. We have yet to decide how we will expose this feature
 // once it is complete.
-BASE_FEATURE(kFluentOverlayScrollbar, base::FEATURE_DISABLED_BY_DEFAULT);
+BASE_FEATURE(kFluentOverlayScrollbar, base::FEATURE_ENABLED_BY_DEFAULT);
 
 // When enabled, scrollbars flash only once when a page is loaded or when they
 // become visible on the viewport instead of flashing after every scroll update.
